app/assets/javascripts/custom.coffee

```diff
@@ -3,17 +3,51 @@ toggleVideo = (state) ->
   div = document.getElementById('youvideo')
   iframe = div.getElementsByTagName('iframe')[0].contentWindow
   div.style.display = if state == 'hide' then 'none' else ''
-  func = if state == 'hide' then 'pauseVideo' else 'playVideo'
-  iframe.postMessage '{"event":"command","func":"' + func + '","args":""}', '*'
-  return
+  if state == 'hide' then jQuery('.youvideo').get(0).stopVideo() 
 
 $(document).ready ->
   $('.play-trailer').on 'click', ->
     toggleVideo 'show'
     $('.moviecard').addClass 'movie-view-trailer'
-    return
+
   $('.back-btn').click ->
     $('.moviecard').removeClass 'movie-view-trailer'
     toggleVideo 'hide'
-    return
+
+  jQuery('#myCarousel').carousel interval: 3000 
+
+  $('#btnadd').click ->
+    $('.add_actor').toggle 'slide'
+  
+  url = window.location.href
+  if (~url.indexOf("?"))
+    url = url.substring(0, url.indexOf('?'))
+  id = url.substring(url.lastIndexOf('/') + 1)
+  token = $('meta[name="csrf-token"]').attr('content')
+  $('#actorsubmit').click ->
+    $.ajax
+      type: 'POST'
+      url: id + '/' + $('#actors_actor_id').val() + '/addActor'
+      headers: 'X-CSRF-Token': document.querySelector('meta[name=csrf-token]').content
+      success: (result) ->
+        window.location.reload()
```

Comment:
If we are reloading page after actor is added successfully then we don't need javascript.